### PR TITLE
PR [TA-4815] Supporter cards widget

### PR DESCRIPTION
**Pull request by bjbrewster**
https://edhdev.atlassian.net/browse/TA-4815

### CODE REVIEW

File: src/components/leaderboards/FitnessLeaderboard/index.js [hunk 1]
```diff
@@ -64,25 +66,35 @@ module.exports = React.createClass({
...
 
-  loadLeaderboard: function() {
-    this.setState({ isLoading: true });
+  getEndpoint: function() {
+    var endpoint;
 
     var props = this.props;
-    leaderboard.find(props.campaignUid, props.type, props.limit, this.loadPages);
+    if (props.country) {
+      if (props.campaignSlug) endpoint = campaigns.leaderboardBySlug.bind(campaigns, props.country, props.campaignSlug);
```
Comment (davidherse) [inline src/components/leaderboards/FitnessLeaderboard/index.js]: Can we add some braces

Comment (davidherse) [inline src/components/leaderboards/FitnessLeaderboard/index.js]: Or `endpoint = props.campaignSlug? campaigns.leaderboardBySlug.bind(campaigns, props.country, props.campaignSlug) : endpoint;`

Comment (davidherse) [inline src/components/leaderboards/FitnessLeaderboard/index.js]: or just `if (props.campaignSlug) { endpoint = campaigns.leaderboardBySlug.bind(campaigns, props.country, props.campaignSlug); };`

File: src/components/leaderboards/Leaderboard/index.js [hunk 2]
```diff
@@ -56,28 +59,37 @@ module.exports = React.createClass({
...
-  loadLeaderboard: function() {
-    this.setState({
-      isLoading: true
-    });
+  getEndpoint: function() {
+    var endpoint;
 
     var props = this.props;
+    if (props.country) {
+      if (props.campaignSlug) endpoint = campaigns.leaderboardBySlug.bind(campaigns, props.country, props.campaignSlug);
```
Comment (davidherse) [inline src/components/leaderboards/Leaderboard/index.js]: braces

File: src/components/supporters/Supporters/index.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    });
+  }, 100),
+
+  getEndpoint: function() {
+    var endpoint;
+
+    var props = this.props;
+    if (props.country) {
+      if (props.campaignSlug) endpoint = campaigns.leaderboardBySlug.bind(campaigns, props.country, props.campaignSlug);
+      if (props.charitySlug)  endpoint = charities.leaderboardBySlug.bind(charities, props.country, props.charitySlug);
```
Comment (davidherse) [inline src/components/supporters/Supporters/index.js]: braces

**Comment by slwen** [comment]
:+1:

**Pull request merged**